### PR TITLE
feat: Error on missing module for `impl Interface for Contract`

### PR DESCRIPTION
**Pull request by kulikthebird**

### CODE REVIEW

File: sylvia-derive/src/querier.rs [hunk 1]
```diff
@@ -157,9 +157,7 @@ where
         let querier = trait_module
             .map(|module| quote! { #module ::sv::Querier })
             .unwrap_or_else(|| quote! { sv::Querier });
-        let bound_querier = contract_module
-            .map(|module| quote! { #module ::sv::BoundQuerier})
-            .unwrap_or_else(|| quote! { sv::BoundQuerier });
+        let bound_querier = quote! { #contract_module ::sv::BoundQuerier };
```
Comment (jawoznia) [inline sylvia-derive/src/querier.rs:160]: Same. as `contract_module` is not an option anymore we can just remove this quote and use these values directly in the main quote

**Review by jawoznia — APPROVED** [review]
Minor changes other than that LGTM

File: sylvia-derive/src/multitest.rs [hunk 2]
```diff
@@ -766,10 +756,7 @@ impl<'a> ImplMtHelpers<'a> {
             &associated_items,
         );
 
-        let contract_module = match contract_module {
-            Some(contract_module) => quote! { #contract_module :: },
-            None => quote! {},
-        };
+        let contract_module = quote! { #contract_module :: };
```
[the remark this answers is not in the document]
Comment (kulikthebird) [reply sylvia-derive/src/multitest.rs]: Done

File: sylvia-derive/src/multitest.rs [hunk 3]
```diff
@@ -91,17 +91,7 @@ impl<'a> ContractMtHelpers<'a> {
...
-                ImplItem::Type(ty) if ty.ident == "Error" => Some(&ty.ty),
-                _ => None,
-            });
-            match associated_error {
-                Some(error) => parse_quote!(#error),
-                // This should never happen as the `interface` macro requires the trait to have an associated `Error` type
-                None => unreachable!(),
-            }
-        } else {
+        let error_type: Type = {
```
[the remark this answers is not in the document]
Comment (kulikthebird) [reply sylvia-derive/src/multitest.rs]: Done

File: sylvia-derive/src/querier.rs (hunk 1 above)
Comment (kulikthebird) [reply sylvia-derive/src/querier.rs]: Done

**Pull request merged**